requirements/requirements.txt

```diff
@@ -1,48 +1,48 @@
 adal==1.2.2               # via msrestazure
-aiohttp==3.6.2
+aiohttp==3.6.2            # via -r /awx_devel/requirements/requirements.in
```

Comment:
Almost none of this (other than the runner version) represents an actual change; it's just a change in the way the tooling generates comments.

Reply:
this still doesn't apply the change to requirements_ansible.txt

I'm fully in favor of applying this, it's an intended improvement.